[DM-2452] Parse device bulk registration files in UTF8 with BOM format - graft y2024 (#2117)
SoftwareAG/c8y-docs: content/change-logs/device-management/cumulocity-10-18-540-136-Bulk-registration-can-not-read-utf8-with-bom.md

---

> date: 2024-05-28

Reviewer: ```suggestion
date: 
```

> title: LWM2M agent bulk registration fails when using an Excel created CSV file

Reviewer 2: ```suggestion
title: LWM2M agent bulk registration no longer fails when using an Excel-generated CSV file
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-1KLUzmqfe
    label: LWM2M
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: DM-2452

> version: 10.18.540.136

Reviewer: ```suggestion
version: 10.18.540.135
```

---

> When an Excel-generated CSV file is submitted to the LWM2M bulk device registration, the process fails due to the inability to parse the UTF-8 with BOM marker present at the beginning of the file. This issue has been addressed, and the LWM2M bulk device registration system now successfully accepts CSV files encoded in UTF-8 with BOM format.

Reviewer 2: ```suggestion
When an Excel-generated CSV file was submitted to the LWM2M bulk device registration, the process failed due to the inability to parse the UTF-8 with a BOM marker present at the beginning of the file. This issue has been addressed, and the LWM2M bulk device registration system now successfully accepts CSV files encoded in UTF-8 with BOM format.
```
